### PR TITLE
Enable-Disable routes for developer and credit-estimation journey

### PR DESCRIPTION
**Pull request by AjinkyaJawalkar**
Setting Env variables to enable-disable routes for  developer and credit-estimation journey

### CODE REVIEW

File: packages/webapp/src/routes/signin/callback.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
   const applicationType = getApplicationType(preAuthenticationRoute)
-
   if (applicationType) {
     const applications = await postJson(`${constants.AZURE_FUNCTION_APP_URL}/getapplications`, {
       contactId: account.idTokenClaims.contactId,
       applicationType
     })
     if (applications?.length > 0) {
-      redirectUrl = applicationType === constants.applicationTypes.ALLOCATION ? constants.routes.DEVELOPER_DEVELOPMENT_PROJECTS : constants.routes.BIODIVERSITY_GAIN_SITES
+      if (applications?.length === 1 && applications[0]?.applicationStatus === 'IN PROGRESS') {
```
Comment (AjinkyaJawalkar) [inline packages/webapp/src/routes/signin/callback.js:15]: Condition / code nesting required

File: packages/webapp/src/routes/__tests__/manage-biodiversity-gains.spec.js
Comment (teddmason) [inline packages/webapp/src/routes/__tests__/manage-biodiversity-gains.spec.js:1]: I don't think this test is sufficient for the route.  You should have at least 2 tests for this change.  One where enableDev is true, then you test the returned html for the presence of the development link/section. And the 2nd test where it is false and you can test for the absence of the development link/section

File: packages/webapp/src/routes/__tests__/signin/callback.spec.js [hunk 2]
```diff
@@ -66,148 +67,107 @@ describe('Signin callback handler', () => {
...
-        const getHandler = callback[0].handler
-        await getHandler({ yar: session, query: {} }, h)
-        expect(redirectArgs[0]).toEqual(constants.routes.DEVELOPER_TASKLIST)
-        expect(auth.authenticate).toHaveBeenCalledTimes(1)
-        done()
-      } catch (err) {
-        done(err)
-      }
-    })
+  it('should redirect to the registrations tasklist when an authenticated registration user has more than one associated applications', done => {
```
Comment (teddmason) [inline packages/webapp/src/routes/__tests__/signin/callback.spec.js:81]: Update message registrations tasklist is incorrect, should be biodiversity gain sites.

Comment (teddmason) [reply packages/webapp/src/routes/__tests__/signin/callback.spec.js:81]: or registrations dashboard

**Review by teddmason — COMMENTED** [review]
Few comments to resolve please.

Also can you find out if the styling of the landowner section on /manage-biodiversity-gains is correct when the development projects is disabled. (the grey background stretches across the screen)

**Review by teddmason — APPROVED** [review]

**Review by teddmason — APPROVED** [review]

**Comment by shaundefra** [comment]
Tested by Leena

**Pull request merged**